docs/creating-and-editing-specialist-document-types.md

```diff
@@ -94,37 +94,68 @@ You'll need to [create the new permission manually](https://docs.publishing.serv
+1. In `publishing-api`:
+   - Add the new values to the field you are amending, in the [specialist_document schema](https://github.com/alphagov/publishing-api/blob/6d5595470bd0e7f3072e06f0113e3ca5514b6e98/content_schemas/formats/shared/definitions/_specialist_document.jsonnet). See [example here](https://github.com/alphagov/publishing-api/pull/2659/files#diff-e427ec772dc2597718b907f2db7772ad580d90452a76ce291114ddd0cfacb289). 
+   - Update the facets in example json with the additional values, as in [this commit](https://github.com/alphagov/publishing-api/pull/2659/files#diff-b1571e1d36f1eb32ee304f39c39fd6d94c3be93a4bf88e56140fdf528a0d2338).
 
-Once approved, this change can be merged and deployed.
+   **NOTE**: You will need to run `bundle exec rake build_schemas` to regenerate schemas after adding the new value(s). 
 
-2. In specialist-publisher, add the new values to the relevant file in the [schema](https://github.com/alphagov/specialist-publisher/tree/main/lib/documents/schemas) directory. See [this](https://github.com/alphagov/specialist-publisher/pull/1899/commits/97c8d713f8e62b0cb8763fe26e1dcf5a0435c12d) commit for an example.
+2. In `specialist publisher`, add the new values to the relevant file in the [schema](https://github.com/alphagov/specialist-publisher/tree/main/lib/documents/schemas) directory. See [this](https://github.com/alphagov/specialist-publisher/pull/1899/commits/97c8d713f8e62b0cb8763fe26e1dcf5a0435c12d) commit for an example.
 
-3. In search-api, amend the value in the relevant schema in the [elasticsearch_types](https://github.com/alphagov/search-api/tree/main/config/schema/elasticsearch_types) directory. See [this](https://github.com/alphagov/search-api/pull/2320/commits/0f29e310581e30707eea7fe8c91063974636dbe2) commit for an example.
+3. In `search-api`, amend the value in the relevant schema in the [elasticsearch_types](https://github.com/alphagov/search-api/tree/main/config/schema/elasticsearch_types) directory. See [this](https://github.com/alphagov/search-api/pull/2320/commits/0f29e310581e30707eea7fe8c91063974636dbe2) commit for an example.
 
-4. Republish the finder, see step 3 in the [Deploy and publish](#Deploy-and-publish) section above. You do not need to reindex search :sweat_smile:
+4. Republish the finder, see step 3 in the [Deploy and publish](#deploy-and-publish) section above. It should suffice to run `search:update_schema` in `search-api`.
 
 # __Editing__ a specialist finder
 
 The [schema](https://github.com/alphagov/specialist-publisher/tree/main/lib/documents/schemas) files that define a specialist document, are also used to configure that document's specialist finder. See [this](https://github.com/alphagov/specialist-publisher/pull/1899/commits/925abc689119138a0e04e17d3610f8ae276773dd) commit for an example.
+
+# __Unpublishing__ a specialist finder
```

Comment:
@baisa I am wondering if this should live in its own documentation (as this whole doc is meant to be about a doc type rather than a finder) but I also feel like having them in the same place is useful...